src/patch/new-reload-events.ts

```diff
@@ -25,14 +25,30 @@ export function buildPatchedReloadEventsFunction(
       });
 
       // MODIFICATIONS
+      const styleAsCompleted = shouldBeStyledAsCompletedEvent(
+        event,
+        patchConfiguration.shouldStylePastDayEvents
+      );
+      const styleAsOngoing = shouldBeStyledAsOngoingEvent(
+        event,
+        patchConfiguration.shouldStyleOngoingEvents
+      );
+
+      if (styleAsCompleted && patchConfiguration.shouldHidePastEvents) {
+        continue;
+      }
+
+      let style = null;
+      if (styleAsCompleted) {
+        style = getCompletedEventStyle();
+      } else if (styleAsOngoing) {
+        style = getOngoingEventStyle();
+      }
```

Comment:
```suggestion
      const style = styleAsCompleted 
        ? getCompletedEventStyle() 
        : styleAsOngoing 
          ? getOngoingEventStyle() 
          : null;
```

Reply:
Legit style preference :)

Personally, I think multiple ternary statements on multiple lines are too hard to read for almost no benefit.